ai_agents/ai_travel_planning_agent/app.py

```
@@ -0,0 +1,214 @@
+
+    session_service = st.session_state.adk_session_service
+    app_name = "ai-travel-planning-agent"
+
+    # Ensure the session exists in the session service
+    try:
+        await session_service.get_session(
+            app_name=app_name,
+            user_id=user_id,
+            session_id=session_id,
+        )
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)